Userland/Libraries/LibC/mallocdefs.h

```diff
@@ -16,13 +16,13 @@
 
 #define PAGE_ROUND_UP(x) ((((size_t)(x)) + PAGE_SIZE - 1) & (~(PAGE_SIZE - 1)))
 
-static constexpr unsigned short size_classes[] = { 8, 16, 32, 64, 128, 256, 504, 1016, 2032, 4088, 8184, 16376, 32752, 0 };
+static constexpr unsigned short size_classes[] = { 16, 32, 64, 128, 256, 496, 1008, 2032, 4080, 8176, 16368, 32752, 0 };
```

Comment:
I might be way of base here, but is there a reason for these to not be powers of 2?

Reply:
Not sure. :man_shrugging:

Reply:
The size classes are chosen to maximize block utilization after the cost of the ChunkedBlock header itself is subtracted from the `mmap()` size.